### ISSUE TITLE
No non-deforming transform option for ImageDataGenerator

### ISSUE THREAD

**Issue by RaffEdwardBAH** [fchollet/keras#809]
The image data generator's flow method always perform some random deformations (translation, rotation, etc), if those features were selected. However, there should be a method that only applies the normalization options (mean, stnd dev, whitening) and ignores the deformations. Otherwise there is no reasonable way to obtain a normalized version of images at test time without also deforming them.

**Comment by wongjingping** [comment]
Hi,
You can just specify 0 or False in the options available when constructing an instance of the ImageDataGenerator class. Defaults to none for those 'deforming' transformations such as translations, flipping and flipping.

From the docs:

`keras.preprocessing.image.ImageDataGenerator(featurewise_center=True,
    samplewise_center=False,
    featurewise_std_normalization=True,
    samplewise_std_normalization=False,
    zca_whitening=False,
    rotation_range=0.,
    width_shift_range=0.,
    height_shift_range=0.,
    shear_range=0.,
    horizontal_flip=False,
    vertical_flip=False)`
